design/ceph/ceph-nfs-ganesha.md

```diff
@@ -115,6 +124,21 @@ spec:
     #    memory: "1024Mi"
     # the priority class to set to influence the scheduler's pod preemption
     priorityClassName:
+
+  security:
+    systemSecurityServicesDaemon:
```

Comment:
Shall we shorten it?
```suggestion
    sssd:
```

Reply:
We could. I don't want to confuse users, but when I google `sssd`, it is the only thing listed in results. Shortening it is probably fine, even if users might not know what sssd is.